packages/solid/tsup.config.js

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from "tsup-preset-solid";
```

Comment:
🤔

Reply:
Using tsup's solid preset to compile solid's jsx correctly as this pr also includes UploadButton and UploadDropzone components